### PR TITLE
Fix segmentation fault in number increment

### PR DESCRIPTION
**Pull request by ueXIMcWizBrdyVEciAlSwxP8aaXP70mO**
Platform: NixOS 20.09
Kernel: Linux 5.4.111 (`x86_64-linux`, glibc 2.31)

This bug is identical in nature to #96---unscaled `sprintf` with wide characters. The commit also retroactively changes the fix in #96 to use the `SIZE` macro from `common/multibyte.h`.

My apologies about the piecemeal change requests. I am discovering these issues as I use the program. For completeness' sake, I checked the other instances of `SPRINTF` for the same bug, and it looks like the one fixed by this PR was the last remaining unscaled one.

### CODE REVIEW

File: common/util.c [hunk 1]
```diff
@@ -253,7 +253,7 @@ v_wstrdup(SCR *sp, const CHAR_T *str, size_t len)
 	if (copy == NULL)
 		return (NULL);
 	MEMCPY(copy, str, len);
-	copy[len] = L('\0');
+	copy[len] = '\0';
```
Comment (ueXIMcWizBrdyVEciAlSwxP8aaXP70mO) [inline common/util.c:256]: This was an unnecessary change on my part, so I reverted it.

Comment (lichray) [reply common/util.c:256]: I actually prefer the way in #96. But for foolish consistency, sure...

**Review by lichray — APPROVED** [review]
Thanks.
